developer-docs-site/docs/nodes/validator-node/using-aws.md

```diff
@@ -66,11 +66,11 @@ Make sure you complete these pre-requisite steps before you proceed:
 
     module "aptos-node" {
       # Download Terraform module from aptos-labs/aptos-core repo
-      source        = "github.com/aptos-labs/aptos-core.git//terraform/aptos-node/aws?ref=main"
+      source        = "github.com/aptos-labs/aptos-core.git//terraform/aptos-node/aws?ref=testnet"
```

Comment:
Probably can remove the double /

Reply:
that's actually required for the path to work